.github/workflows/test_all_in_one_common.yml

```diff
@@ -0,0 +1,90 @@
+name: test-all-in-one-common
+on:
+  workflow_call:
+    inputs:
+      secondary_tests:
+        required: true
+        type: string
+env:
+  UYUNI_PROJECT: uyuni-project
+  UYUNI_VERSION: master
+  CUCUMBER_PUBLISH_TOKEN: ${{ secrets.CUCUMBER_PUBLISH_TOKEN }}
+jobs:
+  test-uyuni:
+    runs-on: ubuntu-latest
+    # create a PR to add yourself if you want to be a beta user and
+    # run this action in your Pull Requests
+    if: github.actor == 'jordimassaguerpla' || github.actor == 'elariekerboull' ||  github.actor == 'srbarrios'
+    steps:
+      - name: welcome_message
+        run: echo "Running acceptance tests. More info at https://github.com/uyuni-project/uyuni/wiki/Running-Acceptance-Tests-at-PR"
+      - name: Install-Podman
+        run: sh -c "sudo apt-get update && sudo apt-get -y install podman"
+      - uses: actions/checkout@v3
+      - name: Cache-jar-files
+        uses: actions/cache@v3
+        with:
+          path: java/buildconf/ivy/repository/
+          key: ${{ runner.os }}-build-cache-uyuni-jars-${{ hashFiles('**/java/buildconf/ivy/*.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-build-cache-uyuni-jars-
+
+      - name: Cache-obs-to-maven files
+        uses: actions/cache@v3
+        with:
+          path: java/.obs-to-maven-cache
+          key: ${{ runner.os }}-build-cache-uyuni-obs-to-maven-${{ hashFiles('**/java/buildconf/ivy/obs-maven-config.yaml') }}
+          restore-keys: |
+            ${{ runner.os }}-build-cache-uyuni-obs-to-maven-
+      - name: Cache-nodejs
+        uses: actions/cache@v3
+        with:
+          path: web/html/src/node_modules
+          key: ${{ runner.os }}-build-cache-uyuni-nodejs-${{ hashFiles('**/web/html/src/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-build-cache-uyuni-nodejs-
+      - name: create_tmp
+        run: ./testsuite/podman_runner/01_setup_tmp_dirs.sh
+      - name: create-podman-network
+        run: ./testsuite/podman_runner/02_setup_network.sh
+      - name: start_controller
+        run: ./testsuite/podman_runner/03_run_controller.sh 
+      - name: create_ssh_conf
+        run: ./testsuite/podman_runner/04_setup_ssh_controller.sh 
+      - name: install_gems_in_controller
+        run: ./testsuite/podman_runner/05_install_gems_in_controller.sh
+      - name: start-server-container
+        run: ./testsuite/podman_runner/06_start_server.sh 
+      - name: mgr-setup
+        run: ./testsuite/podman_runner/07_manager_setup.sh 
+      - name: build_code
+        run: ./testsuite/podman_runner/08_build_server_code.sh 
```

Comment:
While developing having these scripts in independent files was good.
But once this works fine, I suggest to group some of them in a unique script.
As we don't need that much detail of the progress, and if so, we can have the progress in log outputs.

Reply:
Something like https://github.com/uyuni-project/uyuni/blob/master/testsuite/podman_runner/run ?

Reply:
I am calling each step separate because this way in the github action UI you can see all phases separately.